src/Parser/Core/Modules/Items/Cinidaria.js

```diff
@@ -19,16 +18,15 @@ class Cinidaria extends Module {
   }
 
   item() {
-    const fightLengthSec = this.owner.fightDuration / 1000;
     const symbioteStrike = this.abilityTracker.getAbility(SPELLS.SYMBIOTE_STRIKE.id);
-    const dps = symbioteStrike.damageEffective / fightLengthSec;
-    const hps = symbioteStrike.healingEffective / fightLengthSec;
+    const dps = symbioteStrike.damageEffective;
+    const hps = symbioteStrike.healingEffective;
```

Comment:
Naming the variables `damage` and `healing` would make more sense since this is not "per second".